tests/main.cpp

```diff
@@ -57,14 +55,25 @@ TEST_CASE("Simulate & Send Transaction") {
 
 TEST_CASE("Request Airdrop") {
   const solana::Keypair keyPair = solana::Keypair::fromFile(KEY_PAIR_FILE);
-  auto connection = solana::rpc::Connection(solana::DEVNET);
-
-  auto prev_sol = connection.getBalance(keyPair.publicKey);
-  auto result = connection.requestAirdrop(keyPair.publicKey, 1000000000);
-  std::this_thread::sleep_for(std::chrono::seconds(15));
-  auto new_sol = connection.getBalance(keyPair.publicKey);
-  // TODO: validate using confirmTransaction
-  CHECK_GT(new_sol["value"], prev_sol["value"]);
+  const auto connection = solana::rpc::Connection(solana::DEVNET);
+  // request Airdrop
+  const auto prev_sol = connection.getBalance(keyPair.publicKey);
+  const auto signature = connection.requestAirdrop(keyPair.publicKey, 50001);
+  uint64_t timeout = 15;
+  // check signature status
+  // this is a temporary fix. This will be changed to the confirmTransaction
+  // function call once it gets implemented
+  while (timeout >= 0) {
```

Comment:
This will not work the type is uint64_t. timeout will go till 10 and then will go to max of uint64_t and not to -1. 
Please do not use big datatypes if not necessary. timeout is 15 < 256 so short can be used instead.